### PR TITLE
Feature/pelagos 4253 replace consumer with new handler s

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/MessageHandler/DatasetSubmissionFilerHandler.php [hunk 1]
```diff
@@ -153,7 +153,7 @@ private function processFile(File $file): void
         // TODO implement method
 
         try {
-            $newFileDestination = $this->datastore->addFile(fopen($filepath, 'r'));
+            $newFileDestination = $this->datastore->addFile(['fileStream' => fopen($filepath, 'r')]);
```
Comment (mickel1138) [inline src/MessageHandler/DatasetSubmissionFilerHandler.php:156]: Should we turn this into a helper class so it's a little easier?

Comment (praneethpr) [reply src/MessageHandler/DatasetSubmissionFilerHandler.php:156]: Even if its a helper class, the object of the helper class should return resource type inside an array

**Review by mickel1138 — APPROVED** [review]
This part works.

**Review by fightingtexasaggie — APPROVED** [review]
reviewed, tested...seems to work

**Pull request merged**